src/ripple/app/ledger/Ledger.cpp

```diff
@@ -873,19 +875,20 @@ static bool saveValidatedLedger (
 
         std::string const ledgerSeq (std::to_string (seq));
 
-        for (auto const& vt : aLedger->getMap ())
+        for (auto const& [_, acceptedLedgerTx] : aLedger->getMap ())
         {
-            uint256 transactionID = vt.second->getTransactionID ();
+            (void)_;
```

Comment:
Is this due to older GCC not liking unused structured bindings? Do you prefer this over using `[[maybe_unused]]` on the whole structured decomposition to be clear which variable(s) are not used.

Reply:
It appears that neither gcc nor clang are warning about unused structured bindings. I just assumed that they would and pre-emptively added the `(void)_;` statements. I'll remove them.

Reply:
Even better. 👍 

I recall running into issues with unused parts of a structured binding when I first started using C++17 on older compilers last year; IIRC it was only with GCC 6.

Reply:
I only tested on my local system. I'll have to add them back if one of the CI configurations triggers on them. But gcc 7 is the new min version, so if it's only gcc 6 we should be OK.

Reply:
Fixed in `71c328bbc * [fold] Remove "maybe unused" warning work around:`